prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/jdbc/JdbcWorkFlowTask.java

```diff
@@ -0,0 +1,98 @@
+package com.redhat.parodos.tasks.jdbc;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.parodos.workflow.exception.MissingParameterException;
+import com.redhat.parodos.workflow.parameter.WorkParameter;
+import com.redhat.parodos.workflow.parameter.WorkParameterType;
+import com.redhat.parodos.workflow.task.BaseWorkFlowTask;
+import com.redhat.parodos.workflow.task.enums.WorkFlowTaskOutput;
+import com.redhat.parodos.workflows.work.DefaultWorkReport;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class JdbcWorkFlowTask extends BaseWorkFlowTask {
+
+	enum OperationType {
+
+		QUERY, UPDATE, EXECUTE
```

Comment:
What about batch operations? 
Or a complex?
Also, what's the difference between the different operations?
How would it support a need of a customer to either perform 2 update statements or commit non (if one failed)?

Reply:
the requirement is for a single statement that can be anything. You can talk to Luke and Piotr if you think we should support something else. As the requirement says in Jira: CRUD. Not transactions or batches.
My personal opinion is that we obviously support it all. Every SQL server such as Postgres and Mysql supports a ';' separator between queries in a single line. Multi-statement. Including 'auto commit off/on'. User can and should execute the workflow once for each update.

Reply:
If yo think otherwise please suggest a code change.

Reply:
the difference between the operations is described in the README.